mmdet/models/dense_heads/ddod_head.py

```diff
@@ -0,0 +1,796 @@
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule, Scale, bias_init_with_prob, normal_init
+from mmcv.runner import force_fp32
+
+from mmdet.core import (anchor_inside_flags, build_assigner, build_sampler,
+                        images_to_levels, multi_apply, multiclass_nms,
+                        reduce_mean, unmap)
+from ..builder import HEADS, build_loss
+from .anchor_head import AnchorHead
+from mmdet.core.bbox import bbox_overlaps
+
+EPS = 1e-12
+
+@HEADS.register_module()
+class DDODHead(AnchorHead):
```

Comment:
This head is similar to atss head, is it possible to inherit atss head？

Reply:
It will be better not to inherit ATSSHead because DDOD modifies various methods.

Reply:
OK